.github/workflows/build-deploy-forms.yml

```diff
@@ -0,0 +1,56 @@
+name: Build and Deploy Formio Server.
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment the formio is deployed."
+        required: true
+        type: environment
+      formioTag:
+        description: "Formio tag, please refer https://github.com/formio/formio/tags to deploy the appropriate tag."
+        type: string
+        required: true
+        default: "v2.5.3"
+
+jobs:
+  build_deploy:
+    name: Build and Deploy Formio Server.
+    runs-on: ubuntu-latest
+    environment: ${{ github.event.inputs.environment }}
+    env:
+      FORMIO_SOURCE_REPO_TAG: ${{ github.event.inputs.formioTag }}
+      FORMS_NAME: forms
+      BUILD_ID: ${{ github.event.inputs.buildId }}
+      NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
+      BUILD_REF: ${{ github.event.inputs.buildRef }}
+      HOST_PREFIX: ${{ secrets.HOST_PREFIX }}
+      MONGODB_URI: ${{ secrets.MONGODB_URI }}
+      BUILD_NAMESPACE: 0c27fb-tools
+    steps:
+      - name: Print env
+        run: |
+          echo BUILD ID: $BUILD_ID
+          echo BUILD NAMESPACE: $BUILD_NAMESPACE
+          echo NAMESPACE: $NAMESPACE
+          echo BRANCH: $BUILD_REF
+          echo HOST_PREFIX: $HOST_PREFIX
+      # Checkout the PR branch
+      - name: Checkout Target Branch
+        uses: actions/checkout@v1
+      # Log in to OpenShift.
+      # Note: The secrets needed to log in are NOT available if the PR comes from a FORK.
+      # PR's must originate from a branch off the original repo or else all openshift `oc` commands will fail.
+      - name: Log in to OpenShift
+        run: |
+          oc login --token=${{ secrets.SA_TOKEN }} --server=https://api.silver.devops.gov.bc.ca:6443
+      # Build Forms
+      - name: Build Forms
```

Comment:
Following the name, can we please change those to `Build Form Server`? Same for line 53.